torch_glow/tests/utils.py

```diff
@@ -23,17 +31,19 @@ def jitVsGlow_(f_torch, f_glow, *inputs):
 
         # check that there are no Glow nodes in the torch graph
         torch_graph = torch_trace.graph_for(*inputs)
-        print("torch_graph,", torch_graph)
         num_glow_nodes = len(torch_graph.findAllNodes(GLOW_NODE_NAME))
         assert num_glow_nodes == 0, "Expected no Glow nodes, found {}".format(
             num_glow_nodes
         )
 
-        # check that there is exactly 1 Glow node in the glow graph
+        # Check that only ops in expected_fused_ops were fused
```

Comment:
the logic looks right but not the comment
It should be that no expected_fused_ops ops were not fused

I also think we should add a check to make sure that there is at least one of each op in expected_fused_ops in the graph because otherwise the test op may get lowered to something else that isn't fused and there is no warning of it

Reply:
In fact here's an example where this could have helped
https://github.com/pytorch/glow/blob/30df28ebd82cd55ecc9957473558cf62b3244058/torch_glow/tests/nodes/addmm_test.py#L8-L12